compiler/pota-quantization-value-test/CMakeLists.txt

```diff
@@ -92,6 +92,14 @@ add_test(
           ${QUANTIZATION_VALUE_TEST_WITH_PARAM}
 )
 
+add_test(
```

Comment:
Please split `pota-` and `record-` modules.
@binarman , I am keep saying this to SRR members. Why do you not listen to this?